### PR TITLE
Update bioconda-utils to 2.8.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`bioconda-utils`](https://bioconda.github.io/recipes/bioconda-utils/README.html): **2.4.0** &rarr; **2.8.0**

[install with bioconda](http://bioconda.github.io/recipes/bioconda-utils/README.html) [Conda](https://anaconda.org/bioconda/bioconda-utils/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/bioconda-utils`](https://github.com//bioconda/bioconda-recipes/tree/bump/bioconda_utils/recipes/bioconda-utils) (click to view/edit other files)
Summary | Utilities for building and managing bioconda recipes.
Home | [http://bioconda.github.io/build-system.html](http://bioconda.github.io/build-system.html)
Releases |[https://github.com/bioconda/bioconda-utils/tags](https://github.com/bioconda/bioconda-utils/tags)
Author | `@bioconda`
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

**Comment by rpetit3** [comment]
Original error, wondering if its related to 3.12

```
AttributeError: module 'configparser' has no attribute 'SafeConfigParser'
```

**Comment by mencian** [comment]
Seems to be an issue with versioneer's `SafeConfigParser` class, which was removed in Python 3.12.

**Review by bgruening — APPROVED** [review]

**Comment by aliciaaevans** [comment]
Going to try closing this PR and reopening to see if that fixes the timeout issue.

**Comment by aliciaaevans** [comment]
Finally got bioconda-utils to pass the checks on my test branch (https://github.com/bioconda/bioconda-recipes/pull/43796) by removing the test from `meta.yaml`. So the build is succeeding, but there was no way to see that because the docker logs aren't making it to Azure and the pipeline agent gets abandoned.

I'm going to see if I can figure out why that test is causing it to apparently exceed the 7GB of memory or misbehave in some other way (though in my testing on other machines it's only using about 3.56 GB max).

**Comment by aliciaaevans** [comment]
> Original error, wondering if its related to 3.12
> 
> ```
> AttributeError: module 'configparser' has no attribute 'SafeConfigParser'
> ```

@rpetit3 is the `python <3.12` constraint something we need to remove/resolve before merging this?

File: recipes/bioconda-utils/meta.yaml
Comment (aliciaaevans) [inline recipes/bioconda-utils/meta.yaml:90]: I guess mamba decided this wasn't a complete sentence.

File: recipes/bioconda-utils/meta.yaml
Comment (aliciaaevans) [inline recipes/bioconda-utils/meta.yaml:23]: Removed version, so I believe it's now properly using the pinned python version (3.10).

**Comment by bgruening** [comment]
This seems to work. No idea why we are getting this error now. Should we merge to get this release out and try in a build-1 to see why this is failing with an import-test?

**Comment by bgruening** [comment]
@aliciaaevans what do you think?

**Comment by aliciaaevans** [comment]
Good find, @bgruening! Yeah, I think at this point it makes sense to just merge this (once the checks finish) and we can try to figure that error out separately.

**Comment by bgruening** [comment]
Fingers crossed! Thanks @aliciaaevans!

**Pull request merged**
